src/main/java/io/opentracing/contrib/jdbc/JdbcTracingUtils.java

```diff
@@ -68,14 +76,18 @@ static Span buildSpan(String operationName,
 
     final Span span = buildSpan(operationName, sql, connectionInfo, withActiveSpanOnly,
     ignoreStatements, tracer);
-     try (Scope ignored = tracer.activateSpan(span)) {
+    long time = System.nanoTime();
```

Comment:
```suggestion
    long time = slowQueryThresholdMs  > 0 ? System.nanoTime() : 0;
```
don't obtain time if not necessary